interactive-etl/data-generator.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: recommendation-app-data
```

Comment:
should the app name be different? e.g. interactive-etl-data?

Reply:
Well spotted! I will make this distinct.